### ISSUE TITLE
Tabulka "akce" pole "final_cj" - špatná hodnota

### ISSUE THREAD

**Issue by motyc** [ARUP-CAS/aiscr-amcr#76]
U projektových akcí ("typ" = R) má být asi všude final_cj automaticky "PRAVDA". Nyní je to nějak zvláštně. Nepravda logicky musí být jen u samostatných akcí, které mají stále v "ident_cely" X na začátku.

**Comment by pbartosova** [comment]
Podmínka v databázi je stanovena na not null implicitně false. Nelze tu rozlyšovat typ akce. Kde přesně toto nastavení vadí?

**Comment by motyc** [comment]
No, ono to funkčně nevadí, ale je to hrozně matoucí při prohlížení databáze. Nejde to tedy přepisovat nějak ex-post? Ono úplně nejsprávnější u projektových akcí by bylo "null", protože tam je pouze definitivní ID.

**Comment by pbartosova** [comment]
Chápu, ale na druhou stranu se bojím, že když do toho sáhnu, rozbije se někde něco úplně jiného. Každopádně to zkontroluji.

**Comment by motyc** [comment]
Ok, můžeme si to tu nechat poznamenané a řešit v budoucnu. Není to tak podstatné.
